pilot/ComposeViewController.swift

```diff
@@ -96,21 +96,25 @@ class ComposeViewController: UIViewController {
     self.present(gallery, animated: true, completion: nil)
   }
 
-  @objc func cancel(_ sender: UIBarButtonItem) {
+  @objc func cancelAction(_ sender: UIBarButtonItem) {
     self.textView.resignFirstResponder()
     self.resignFirstResponder()
     self.dismiss(animated: true, completion: nil)
   }
 
-  @objc func post(_ sender: UIBarButtonItem) {
-    // Send the post to Lightning
+  @objc func nextAction(_ sender: UIBarButtonItem) {
+    // Push the SelectionViewController onto the navigations stack
+    let selectionViewController = SelectionViewController()
+    selectionViewController.post = self.postState
```

Comment:
How does `self.postState` get set?

Reply:
ComposeViewController sets the state before switching:

    let selectionViewController = SelectionViewController()
    selectionViewController.post = self.postState

    self.navigationController?.pushViewController(selectionViewController, animated: true)